app/javascript/components/top-corner-controller.jsx

```diff
@@ -37,7 +37,7 @@ const TopCorner = ({ showWeather }) => {
       </IconWrapper>
       <Row>
         <TimeHero />
-        {!showWeather && <WeatherCorner />}
+        {showWeather && <WeatherCorner />}
```

Comment:
:thinking: looks like this is a bug fix from the previous commit. I think changing the design to properly fit when always displaying the weather makes sense as a single commit.